docs/index.md

```diff
@@ -144,3 +144,4 @@ After successfully installing Codacy on your cluster, you are now ready to perfo
 3.  As a last step we recommend that you [set up monitoring](configuration/monitoring.md) on your Codacy instance.
 
 If you run into any issues while configuring Codacy, be sure to [check our troubleshooting guide](troubleshoot/troubleshoot.md) for more help.
+If you with to setup HTTPS ingress for Codacy on AWS please see see our [installation guide](configuration/tls-ingress-aws.md).
```

Comment:
```suggestion
3.  If you are using Amazon EKS as your Kubernetes infrastructure and you wish to set up HTTPS ingress for Codacy, follow our [configuration instructions](configuration/tls-ingress-aws.md).

4.  As a last step we recommend that you [set up monitoring](configuration/monitoring.md) on your Codacy instance.

If you run into any issues while configuring Codacy, be sure to [check our troubleshooting guide](troubleshoot/troubleshoot.md) for more help.
```

Reply:
What do you think of being a bit more prescriptive and including this configuration as part of the "post-install configuration steps", even though it's up to the customer to decide if they want to have HTTPS or not?

Reply:
Can we also make this work with MicroK8s on Amazon EKS? We would need to instruct using `microk8s.kubectl` instead of `kubectl` on the appropriate commands.

Reply:
Initially I left it out of the basic setup because it has a few moving parts, it isn't something everyone will need, and users might want to setup their certificates differently. But your suggested edit seems fine because it conveys the message that this is not a requirement.

Regarding microk8s support, we should do it, but I think it is better to split this into two tasks to unblock users on AWS that currently require this. Afterwards we can broaden the scope of the docs to encompass microk8s as well. I don't think it will require much work, I just haven't tested this sufficiently there. WDYT?

Reply:
It makes total sense, let's release this now :rocket: and work on the configuration MicroK8s later. I'm hopeful that the configuration for MicroK8s won't be too different from this one.